source/type/type.h

```diff
@@ -619,13 +637,14 @@ struct Line : public Header, Nameable, HasMessages {
     std::map<std::string,std::string> properties;
 
     std::vector<LineGroup*> line_group_list;
+    Contributor* contributor=nullptr;
```

Comment:
Just to note it for later, stocking the contributor for the line depends on the handling of the Frame at the vj level